include/CppUTest/StandardCLibrary.h

```diff
@@ -1,8 +1,10 @@
 
+/* Must include this first to ensure the StandardC include in CppUTestConfig still happens at the right moment */
+#include "CppUTestConfig.h"
```

Comment:
As long as there is no clear problem but a clear benefit not.

If it can be fixed without losing it benefit, then I might entertain it :)